prdoc/pr_4118.prdoc

```diff
@@ -0,0 +1,13 @@
+title: "pallet assets: minor improvement on errors returned for some calls"
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Some calls in pallet assets have better errors. No new error is introduced, only more sensible choice are made.
+  - audience: Runtime User
+    description: |
+      Some calls in pallet assets have better errors. No new error is introduced, only more sensible choice are made.
```

Comment:
I feel information is not useful, but in theory it impacts both dev for maybe test to udpate and frontend.

Reply:
in theory it is kind of a major breaking change :-(